mithril-client/Dockerfile.ci

```diff
@@ -3,15 +3,20 @@
 # on a debian-compatible x86-64 environment
 FROM debian:11-slim
 
-# Upgrade
-RUN apt-get update -y && apt-get install -y libssl-dev ca-certificates wget && rm -rf /var/lib/apt/lists/*
-
 # Create appuser
 RUN adduser --disabled-password appuser
 
+# Precreate workdir
+RUN mkdir -p /app/bin
+
+# Upgrade
+RUN apt-get update -y && apt-get install -y libssl-dev ca-certificates wget && apt-get clean && rm -rf /var/lib/apt/lists/*
+
 # Copy the executable
 COPY mithril-client/mithril-client /app/bin/mithril-client
-COPY mithril-client/config /app/config
+
+# Copy the config files
+COPY config /app/config
```

Comment:
Same problem as above:
```suggestion

# Copy the config files
COPY mithril-client/config /app/config
```

Reply:
Yep, I saw the failure and already adjusted for it. New checks should kick off after workflow approval